src/js/components/user-management/selfusermanagement.js

```diff
@@ -189,6 +189,7 @@ const mapStateToProps = state => {
   return {
     has2FA: state.users.globalSettings.hasOwnProperty('2fa') && state.users.globalSettings['2fa'] === 'enabled',
     isEnterprise: state.app.features.isEnterprise || (state.app.features.isHosted && plan !== 'os'),
```

Comment:
this one I dont get as well, if the plan is not OS  we set isEnterprise to true while on hosted, are we loosing Professional here? which isHosted, is not enterprise, and is not os